cmd/contour/bootstrap.go

```diff
@@ -35,5 +35,6 @@ func registerBootstrap(app *kingpin.Application) (*kingpin.CmdClause, *envoy.Boo
 	bootstrap.Flag("envoy-key-file", "Client key filename for Envoy secure xDS gRPC communication.").Envar("ENVOY_KEY_FILE").StringVar(&config.GrpcClientKey)
 	bootstrap.Flag("namespace", "The namespace the Envoy container will run in.").Envar("CONTOUR_NAMESPACE").Default("projectcontour").StringVar(&config.Namespace)
 	bootstrap.Flag("xds-resource-version", "The versions of the xDS resources to request from Contour.").Default("v3").StringVar((*string)(&config.XDSResourceVersion))
+	bootstrap.Flag("dns-lookup-family", "DNS Resolution Policy. Either v4, v6 or auto").StringVar(&config.DNSLookupFamily)
```

Comment:
This addition also needs some docs here: https://projectcontour.io/docs/v1.12.0/configuration/#flags (source file https://github.com/projectcontour/contour/blob/d31d2b6fe22097e44ab55924e4707927c5084807/site/docs/main/configuration.md#flags)

Reply:
Thanks @sunjayBhatia . Fixed this in next patchset.